src/test/scala/com/gu/commercial/display/AdTargeterSpec.scala

```diff
@@ -13,21 +14,21 @@ class AdCallSpec extends FlatSpec with Matchers with OptionValues {
     }
   }
 
-  private val adCall = new AdCall(platform = "ng", surgeLookup)
+  private val targeter = new AdTargeter(platform = "ng", surgeLookup)
 
-  private def stringify(params: Map[AdCallParamKey, AdCallParamValue]) =
-    params.map {
-      case (k, v) =>
-        k.name -> (v match {
-          case sv: SingleValue    => sv.value
-          case mv: MultipleValues => mv.values
-        })
+  private def stringify(params: Set[AdTargetParam]): Map[String, Any] =
```

Comment:
This doesn't do what I would expect; although I can see why it's called `stringify`. But even then, it doesn't convert all values of the `Map` to a `String`, it also contains `Set`s?